examples/hello_android/app/src/main/java/com/example/flutter/ExampleActivity.java

```diff
@@ -76,6 +77,16 @@ protected void onResume() {
         flutterView.onResume();
     }
 
+    @Override
+    protected void onNewIntent(Intent intent) {
+        // Reload the Flutter Dart code when the activity receives an intent
+        // from the "flutter refresh" command.
+        if (Intent.ACTION_RUN.equals(intent.getAction())) {
```

Comment:
I doubt we want this ability in production code.  Unclear if this should use ACTION_RUN or not:
http://developer.android.com/reference/android/content/Intent.html#ACTION_RUN

Seems like any other process on the phone could hijack the permissions of this app this way. @abarth